src/Fixer/Operator/BinaryOperatorSpacesFixer.php

```diff
@@ -760,7 +760,7 @@ private function replacePlaceholders(Tokens $tokens, $alignStrategy)
                             }
                         }
 
-                        $lines[$index] = $before.substr($lines[$index], $currentPosition);
+                        $lines[$index] = $before.mb_substr($lines[$index], $currentPosition);
```

Comment:
We don't require `mb_string` extension nor a polyfill for it (we only suggest it in the composer.json). Therefore this might not work for everyone. 
Not sure if we want to start requiring it here now, can we work around it in a reasonable matter?